docs/digger/developer.asciidoc

```diff
@@ -5,4 +5,20 @@ title: Digger developer guide
 toc_generate: true
 ---
 
-include::docs/digger/developer/jenkinsfile.asciidoc[Using Jenkinsfiles]
+== Overview
+
+include::docs/digger/developer/jenkinsfile-overview.asciidoc[]
+
+== Anatomy of a Jenkinsfile
+
+include::docs/digger/developer/jenkinsfile-anatomy.asciidoc[]
+
+== Using Jenkinsfiles in an app
+
+include::docs/digger/developer/jenkinsfile-create.asciidoc[]
+
+=== Jenkins pipeline documentation
```

Comment:
let's follow the same convention and put the titles inside each of the document. It will make it easier to manage the section titles. Here we can add some comments.